packages/search-ui-app-search-connector/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
do you know what triggered this? AFAIK, we dont and shouldn't have any JSX in this connector

Reply:
Accidentally added it